README.md

```diff
@@ -44,11 +42,11 @@ You can integrate `TwilioVideo.framework` manually by following these [install i
 
 ## Quickstart
 
-### Running the Quickstart
+### Running the Swift Quickstart
```

Comment:
`QuickStart` - and below as well

Reply:
The other quickstart repos are using `Quickstart`. Leaving as is for the time being unless we want to change it across all quickstart repos.